src/System.Data.SqlClient/tests/FunctionalTests/System.Data.SqlClient.Tests.builds

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
My plan is to land this change and do another pass which cleans up the .builds stuff and cleans up the traversal stuff.